examples/nextjs-swc/package.json

```diff
@@ -7,13 +7,13 @@
     "debug": "NODE_OPTIONS='--inspect' next dev",
     "build": "yarn lang:extract && next build",
     "start": "next start",
-    "lang:extract": "lingui extract --clean",
+    "lingui:extract": "lingui extract --clean",
     "test": "yarn build"
   },
   "dependencies": {
     "@lingui/core": "^4.1.2",
     "@lingui/react": "^4.1.2",
-    "next": "13.2.3",
+    "next": "13.3.1",
```

Comment:
The release version of `@lingui/swc-plugin` was built with a newer version of the `swc-core`, which is not compatible with `next@13.2.3"` this caused a "panic" during build. I put a compatible version and added a compatibility notice into readme.